ha/cli/exec/clusterExecutor.py

```diff
@@ -15,118 +15,34 @@
-
+        raise HAUnimplemented("This operation is not implemented.")
 
     def execute(self):
-
-        # This is temporary code, copied from M0
-        # To be removed once the "cortx cluster start" user story [EOS-16248] is started
+        """
+        Execute cluster start operation
+        """
         Log.info("Executing cortxha cluster start")
-        print("Executing cortxha cluster start")
-
-        _output, _err, _rc = self._execute.run_cmd(const.PCS_CLUSTER_STATUS, check_error=False)
-        if _rc != 0:
-            if(_err.find("No such file or directory: 'pcs'") != -1):
-                Log.error("Cluster failed to start; pcs not installed")
-                #print("Cluster failed to start; pcs not installed")
-                raise HAClusterStart("Cluster failed to start; pcs not installed")
-            # if cluster is not running; start cluster
-            elif(_err.find("cluster is not currently running on this node") != -1):
-                self._execute.run_cmd(const.PCS_CLUSTER_START, check_error=False)
-                Log.info("cluster started ; waiting for nodes to come online ")
-                # It takes nodes 30 seconds to come to their original state after cluster is started
-                # observation on a 2 node cluster
-                # wait for upto 100 sec for nodes to come to active states (online / maintenance mode)
-                time.sleep(10)
-                self.get_nodes_status()
-                retries = 18
-                while  self.active_nodes == False and retries > 0:
-                    time.sleep(5)
-                    self.get_nodes_status()
-                    retries -= 1
-
-        else:
-            #If cluster is running, but all nodes are  in Standby mode;
-            #start the nodes
-            self.get_nodes_status()
-            if self.active_nodes == False:
-                if self.standby_nodes == True:
-                    # issue pcs cluster unstandby
-                    _output, _err, _rc = self._execute.run_cmd(const.PCS_CLUSTER_UNSTANDBY, check_error=False)
-
-        # check cluster and node status
-        _output, _err, _rc = self._execute.run_cmd(const.PCS_CLUSTER_STATUS, check_error=False)
-        if _rc != 0:
-            # cluster could not be started.
-            Log.error("Cluster failed to start")
-            raise HAClusterStart("Cluster failed to start")
-        else:
-            # confirm that at least one node is active
-            self.get_nodes_status()
-            if self.active_nodes == False:
-                # wait for 5 seconds and retry
-                time.sleep(5)
-                self.get_nodes_status()
-                if self.active_nodes == False:
-                    Log.info("Cluster started; nodes not online")
-                    raise HAClusterStart("Cluster started; nodes not online")
-
-        Log.info("Cluster started successfully")
+        result = self._controllers[const.CLUSTER_CONTROLLER].start()
```

Comment:
No code here. It should have all the logic implemented earlier.

Reply:
Added logic